src/Components/Artwork/Save.tsx

```diff
@@ -44,7 +44,7 @@ export class SaveButton extends React.Component<SaveProps, SaveState> {
   }
 
   state = {
-    is_saved: null,
+    is_saved: this.props.artwork.is_saved,
```

Comment:
Initially, the artwork button would be rendered in the incorrect state, mostly b/c this wasn't initially set. But there might be something else slightly missing or not hooked up since this is the new custom render stuff....seems to work?

Also, I think you had the SVG flipped, the `selected` state was just the outline, and the unselected state was filled in.

Reply:
Welp, this actually breaks saving for the regular usage tho. Something's up with the custom rendering.

Reply:
Nvm figured it out!! 🦆